Library/Homebrew/cleanup.rb

```diff
@@ -213,6 +219,7 @@ def cleanup_cache(entries = nil)
       entries ||= [cache, cache/"Cask"].select(&:directory?).flat_map(&:children)
 
       entries.each do |path|
+        FileUtils.chmod_R 0755, path if path.needs_chowning? && !dry_run?
```

Comment:
Does this need to be recursive? Could `+r` be used instead? Otherwise this may overwrite a decision to e.g. make this group writable.

Reply:
The cache wouldn't remove locally unless I ran recursively through everything; you end up with the build cache removed but the module cache being persistent. Which isn't the worst thing in the world, a longer-lasting cache is the most effective way for the Go changes to be useful, but I was planning to achieve that a little more legitimately than simply making it impossible for cleanup to remove without user intervention, heh.

> Otherwise this may overwrite a decision to e.g. make this group writable.

Where would that decision come from? I didn't think we especially supported user intervention in cache management?